services/forest/src/util/restAPI.js

```diff
@@ -0,0 +1,19 @@
+const fetch = require('node-fetch');
+const config = require('config');
+
+const backendURL = config.services.backend;
+
+const makeGetRequest = endpoint => (
+  fetch(`${backendURL}/${endpoint}`)
+    .then(response => (response.status === 200 ? response.json() : new Error('Response code is not 200')))
+    .catch(error => new Error('Error making the request', error))
```

Comment:
Again, this messages are for the user.

In the first case I think the error message could be the same sent by the other service (we should check this in future requests)

In the second case could be something like "Error communicating with other services" or "Internal services error" or something like that

Reply:
Also, since we want to log more details you could throw an object with "stack" and "message" properties, so stack can be the service error (the one that will be logged) and message will be the user friendly error

Reply:
Done